src/driver/sqljs/SqljsQueryRunner.ts

```diff
@@ -0,0 +1,99 @@
+import {ObjectLiteral} from "../../common/ObjectLiteral";
+import {QueryRunnerAlreadyReleasedError} from "../../error/QueryRunnerAlreadyReleasedError";
+import {OrmUtils} from "../../util/OrmUtils";
+import {InsertResult} from "../InsertResult";
+import {AbstractSqliteQueryRunner} from "../sqlite-abstract/AbstractSqliteQueryRunner";
+import {SqljsDriver} from "./SqljsDriver";
+
+/**
+ * Runs queries on a single sqlite database connection.
+ *
+ * Does not support compose primary keys with autoincrement field.
+ * todo: need to throw exception for this case.
+ */
+export class SqljsQueryRunner extends AbstractSqliteQueryRunner {
+    
+    /**
+     * Database driver used by connection.
+     */
+    driver: SqljsDriver;
+    
+    // -------------------------------------------------------------------------
+    // Constructor
+    // -------------------------------------------------------------------------
+
+    constructor(driver: SqljsDriver) {
+        super(driver);
+        this.driver = driver;
+        this.connection = driver.connection;
+    }
+
```

Comment:
public methods are separate block, please try to keep following common code convention